ballerina/tests/test.bal

```diff
@@ -0,0 +1,316 @@
+// Copyright (c) 2024 WSO2 LLC. (https://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/os;
+import ballerina/http;
+
+configurable boolean isLiveServer = os:getEnv("IS_LIVE_SERVER") == "true";
+configurable string serviceUrl = isLiveServer ? os:getEnv("SERVICE_URL") : "http://localhost:9090";
+configurable string token = isLiveServer ? os:getEnv("JIRA_TOKEN") : "test";
+configurable string username = isLiveServer ? os:getEnv("JIRA_USERNAME") : "test@example.com";
+ConnectionConfig config = {
+    auth: <http:CredentialsConfig>{
+        username: username,
+        password: token
+    }
+};
+final Client jira = check new Client(config, serviceUrl);
+
+@test:Config {
+    groups: ["live_tests", "mock_tests"]
+}
+function testGetMyself() returns error? {
+    User response = check jira->/rest/api/'3/myself;
+
+    // Assertions
+    test:assertNotEquals(response,(),"Myself data cannot be null");
```

Comment:
```suggestion
    test:assertNotEquals(response, (), "Myself data cannot be null");
```
Rather than checking if the value is not null, shall we assert a field(ex: name) in the response.

Reply:
Yeah will make that change. Can I know why we should assert a specific field?? I mean is there any reason behind that ?